src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -1082,6 +1082,9 @@ public static int ConvertToUtf32(string s, int index)
         // IBinaryInteger
         //
 
+        /// <inheritdoc cref="IBinaryInteger{TSelf}.DivRem(TSelf, TSelf)" />
+        public static (char Quotient, char Remainder) DivRem(char left, char right) => ((char, char))Math.DivRem(left, right);
```

Comment:
It feels weird to me that we're having `char` be a number.  `char` values don't inherently have a notion of addition/subtraction/etc (while the operators work, they leave the domain, and they only work because they convert char to int... so char * char => int).

Reply:
That's no different from `short * short => int` in C#. In other languages, such as F#, `short op short => short` and `char op char => char`.

`char` is functionally "numeric like" and languages provide operator support on it, with users utilizing that number support for various operations/considerations. This was what got taken to API review and how it gets used internally for things like text transcoding algorithms and the like.